pincer/objects/guild/member.py

```diff
@@ -36,8 +36,8 @@ class BaseMember(APIObject):
     hoisted_role: APINullable[:class:`~pincer.utils.snowflake.Snowflake`]
         The user's top role in the guild.
     """
-    joined_at: Timestamp
-    roles: List[Snowflake]
+    joined_at: Timestamp = None
+    roles: List[Snowflake] = None
```

Comment:
Does the API send null as these? If so, see below.
```suggestion
    joined_at: Optional[Timestamp] = None
    roles: Optional[List[Snowflake]] = None
```

If not, 
```suggestion
    joined_at: APINullable[Timestamp] = MISSING
    roles: APINullable[List[Snowflake]] = MISSING
```